### PR TITLE
#420 Completes canonical links task

### LINKED ISSUE #420: Search engine results show older versions as top result
**Summary:**

~~Explore approaches to implement redirect docs links from search engine results pages to latest versions~~

~~For example, Clicking on https://ranchermanager.docs.rancher.com/v2.5/pages-for-subheaders/backup-restore-and-disaster-recovery from a search engine results page should automatically take you to https://ranchermanager.docs.rancher.com/pages-for-subheaders/backup-restore-and-disaster-recovery~~

When using search engines to find documentation, sometimes an older version of a doc will be the top result of a search instead of the latest. Behavior varies with the search term used.

E.g. (Mar 2, 2023) Searching `rancher backup restore` gives https://ranchermanager.docs.rancher.com/v2.6/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-rancher as the top result even though there is https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-rancher available.

### PR DESCRIPTION
**Pull request by martyav**
Fixes #420 

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, make sure to target the release branch instead of `main`.

## Description

This PR covers the 200 files that lacked canonical links, in advance of the v2.8 release, where we'll be updating the /docs directory to match the contents of versioned_docs/version-2.8

### CODE REVIEW

File: docs/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Role-based Access Control
 ---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring"/>
+</head>
```
Comment (martyav) [inline docs/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md:7]: ### 
```suggestion
</head>

```

File: versioned_docs/version-2.6/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Role-based Access Control
 ---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring"/>
+</head>
```
Comment (martyav) [inline versioned_docs/version-2.6/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md:7]:
```suggestion
</head>

```

File: versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Role-based Access Control
 ---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring"/>
+</head>
```
Comment (martyav) [inline versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md:7]:
```suggestion
</head>

```

File: versioned_docs/version-2.8/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Role-based Access Control
 ---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring"/>
+</head>
```
Comment (martyav) [inline versioned_docs/version-2.8/integrations-in-rancher/monitoring-and-alerting/rbac-for-monitoring.md:7]:
```suggestion
</head>

```

**Review by btat — APPROVED** [review]

**Pull request merged**